tiled/_tests/test_catalog.py

```diff
@@ -527,3 +527,34 @@ async def test_constraints_on_parameter_and_num(a, assets):
                 )
             ],
         )
+
+
+@pytest.mark.asyncio
+async def test_init_db_logging(tmpdir, caplog):
+    config = {
+        "database": {
+            "uri": "sqlite+aiosqlite://",  # in-memory
+        },
+        "trees": [
+            {
+                "tree": "catalog",
+                "path": "/",
+                "args": {
+                    "uri": f"sqlite+aiosqlite:///{tmpdir}/catalog.db",
+                    "writable_storage": str(tmpdir / "data"),
+                    "init_if_not_exists": True,
+                },
+            },
+        ],
+    }
+    # Issue 721 notes that the logging of the subprocess that creates
+    # a database logs normal things to error. This test looks at the log
+    # and fails if an error logg happens. This could catch anything that is
+    # and error during the app build.
```

Comment:
```suggestion
    # and fails if an error log happens. This could catch anything that is
    # an error during the app build.
```

Reply:
Thanks. I fixed comments and squashed into a single commit.